### PR TITLE
DOC: Added config options doc

### PR DESCRIPTION
**Pull request by AakashGfude**

### CODE REVIEW

File: docs/source/config.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Configuration options
+=====================
+
+The configuration options specific to ``tomyst`` which you can use in your ``conf.py`` file are:
+
+#. **tomyst_static_file_path**: An array which takes path to your static folders, relative to the ``confdir``.
```
Comment (mmcky) [inline docs/source/config.rst:6]: thanks @AakashGfude this is really the `target` file path right?

sphinx will (by default) copy the `_static` folder to the root level of the target _build folder. This setting moves that location to match the source location (and isn't specifically used to fetch files)

**Review by mmcky — CHANGES_REQUESTED** [review]
@AakashGfude this needs a few more explaining words + an example perhaps to make how to use this `config` clear. Think like you've never used the project before `:-)`

File: docs/source/config.rst (hunk 1 above)
Comment (AakashGfude) [reply docs/source/config.rst:6]: yes, that's true. You are right, I should be more verbose and clear about it.

**Comment by AakashGfude** [comment]
> @AakashGfude this needs a few more explaining words + an example perhaps to make how to use this `config` clear. Think like you've never used the project before `:-)`

have pushed some edits. does it make more sense now?

**Comment by mmcky** [comment]
thanks @AakashGfude that is a big improvement.

**Pull request merged**